pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumbgppeeringpolicies.yaml

```diff
@@ -0,0 +1,163 @@
+                      properties:
+                        key:
+                          description: key is the label key that the selector applies
+                            to.
+                          type: string
+                        operator:
+                          description: operator represents a key's relationship to
+                            a set of values. Valid operators are In, NotIn, Exists
+                            and DoesNotExist.
+                          enum:
+                          - In
+                          - NotIn
+                          - Exists
+                          - DoesNotExist
+                          type: string
+                        values:
+                          description: values is an array of string values. If the
+                            operator is In or NotIn, the values array must be non-empty.
+                            If the operator is Exists or DoesNotExist, the values
+                            array must be empty. This array is replaced during a strategic
+                            merge patch.
+                          items:
+                            type: string
+                          type: array
+                      required:
+                      - key
+                      - operator
+                      type: object
+                    type: array
+                  matchLabels:
+                    additionalProperties:
+                      description: MatchLabelsValue represents the value from the
+                        MatchLabels {key,value} pair.
+                      maxLength: 63
+                      pattern: ^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$
+                      type: string
+                    description: matchLabels is a map of {key,value} pairs. A single
+                      {key,value} in the matchLabels map is equivalent to an element
+                      of matchExpressions, whose key field is "key", the operator
+                      is "In", and the values array contains only "value". The requirements
+                      are ANDed.
+                    type: object
+                type: object
+              virtualRouters:
+                description: A list of CiliumBGPVirtualRouter(s) which instructs the
+                  BGP control plane how to instantiate virtual BGP routers.
+                items:
+                  description: CiliumBGPVirtualRouter defines a discrete BGP virtual
+                    router configuration.
+                  properties:
+                    exportPodCIDR:
+                      description: ExportPodCIDR determines whether to export the
+                        Node's private CIDR block to the configured neighbors.
+                      type: boolean
+                    localASN:
+                      description: LocalASN is the ASN of this virtual router
+                      maximum: 65536
```

Comment:
gobgp advertises the four-octet ASN capability by default. So, should be pow(2, 32) - 1?

Reply:
Yup! Forgot about extended ASN. Good catch. Fix incoming.

Reply:
fixed.